docs/settings.rst

```diff
@@ -35,13 +35,19 @@ Accepts a ``List[ApplyRule]`` which sets the rules to use.
 
     Although ``SCRAPY_POET_RULES`` already has values set from the return value of
     :meth:`web_poet.default_registry.get_rules() <web_poet.rules.RulesRegistry.get_rules>`,
-    make sure you call :func:`web_poet.consume_modules <web_poet.rules.consume_modules>`
-    inside your ``settings.py`` file if you're using other rules from other external
-    packages which contain page objects annotated with ``@handle_urls``.
+    make sure to also set the ``SCRAPY_POET_MODULES`` setting below.
 
 There are sections dedicated for this at :ref:`intro-tutorial` and
 :ref:`rules-from-web-poet`.
 
+SCRAPY_POET_MODULES
+-------------------
+
+Default: ``[]``
+
+A list of modules (i.e. ``List[str]``) which scrapy-poet will look for page objects
+annotated with the :func:`web_poet.handle_urls` decorator. Each module is passed
```

Comment:
I think we should mention 2 things here:

1. It may not be just modules, but packages as well. All modules in a package are loaded recursively.
2. It is possible to get page ojects in the registry even if the module is not in SCRAPY_POET_MODULES. We may say that for handle_urls to work a module needs to be imported, and SCRAPY_POET_MODULES just makes sure that all modules/packages listed there are imported.

Reply:
Added more details in f16dbf2